File: src/1672-Richest-Customer-Wealth/richest_customer_wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
  var max_wealth = 0
  accounts.forEach((customer) => {
      customer_wealth = 0
      customer.forEach((bank) => {
          customer_wealth += bank
      })
      max_wealth = Math.max(max_wealth, customer_wealth)
  })
  return max_wealth
};

accounts = [[1,2,3],[3,2,1]]
console.log(maximumWealth(accounts))

File: src/1672-Richest-Customer-Wealth/go/richest_customer_wealth.go
package main

import "fmt"

func maximumWealth(accounts [][]int) int {
	max := 0

	for i := 0; i < len(accounts); i++ {
		currWealth := 0
		for j := 0; j < len(accounts[i]); j++ {
			currWealth += accounts[i][j]
		}
		if currWealth > max {
			max = currWealth
		}
	}

	return max
}

func main() {
	accounts := [][]int{{1, 2, 3}, {3, 2, 1}}
	res := maximumWealth(accounts)

	fmt.Println(res)
}
